example/src/main/java/io/netty/example/stomp/websocket/StompChatHandler.java

```diff
@@ -0,0 +1,199 @@
+        String destination = inboundFrame.headers().getAsString(DESTINATION);
+        String subscriptionId = inboundFrame.headers().getAsString(ID);
+
+        if (destination == null || subscriptionId == null) {
+            sendErrorFrame("missed header", "Required 'destination' or 'id' header missed", ctx);
+            return;
+        }
+
+        Set<StompSubscription> subscriptions = chatDestinations.get(destination);
+        if (subscriptions == null) {
+            subscriptions = new HashSet<StompSubscription>();
+            Set<StompSubscription> previousSubscriptions = chatDestinations.putIfAbsent(destination, subscriptions);
+            if (previousSubscriptions != null) {
+                subscriptions = previousSubscriptions;
+            }
+        }
+
+        final StompSubscription subscription = new StompSubscription(subscriptionId, destination, ctx.channel());
+        if (subscriptions.contains(subscription)) {
+            sendErrorFrame("duplicate subscription",
+                           "Received duplicate subscription id=" + subscriptionId, ctx);
+            return;
+        }
+
+        subscriptions.add(subscription);
+        ctx.channel().closeFuture().addListener(new ChannelFutureListener() {
+            @Override
+            public void operationComplete(ChannelFuture future) {
+                chatDestinations.get(subscription.destination()).remove(subscription);
+            }
+        });
+
+        String receiptId = inboundFrame.headers().getAsString(RECEIPT);
+        if (receiptId != null) {
+            StompFrame receiptFrame = new DefaultStompFrame(StompCommand.RECEIPT);
+            receiptFrame.headers().set(RECEIPT_ID, receiptId);
+            ctx.writeAndFlush(receiptFrame);
+        }
+    }
+
+    private void onSend(ChannelHandlerContext ctx, StompFrame inboundFrame) {
+        String destination = inboundFrame.headers().getAsString(DESTINATION);
+        if (destination == null) {
+            sendErrorFrame("missed header", "required 'destination' header missed", ctx);
+            return;
+        }
+
+        Set<StompSubscription> subscriptions = chatDestinations.get(destination);
+        for (StompSubscription subscription : subscriptions) {
+            subscription.channel().writeAndFlush(transformToMessage(inboundFrame, subscription));
+        }
+    }
+
+    private void onUnsubscribe(ChannelHandlerContext ctx, StompFrame inboundFrame) {
+        String subscriptionId = inboundFrame.headers().getAsString(SUBSCRIPTION);
+        for (Entry<String, Set<StompSubscription>> entry : chatDestinations.entrySet()) {
+            Iterator<StompSubscription> iterator = entry.getValue().iterator();
+            while (iterator.hasNext()) {
+                StompSubscription subscription = iterator.next();
+                if (subscription.id().equals(subscriptionId) && subscription.channel().equals(ctx.channel())) {
```

Comment:
hmm... shouldn't this also take the destination somehow into account to match `StompSubscription.equals(...)` ?

Reply:
By spec only `id` required for `UNSUBSCRIBE` command.

Reply:
chat destination There is a fuzzy match;
Example: 
chat client destination: /topic/test/*
Whether the server can adapt better

Reply:
@amizurov @normanmaurer 
After the test found that the client port is not detected by the server

Reply:
> chat destination There is a fuzzy match;
> Example:
> chat client destination: /topic/test/*
> Whether the server can adapt better
> @amizurov @normanmaurer

Just for clarify, this is only example how you can integrate (WS and subProtocol(STOMP)). So fuzzy match is business feature if you think that is really needed you can provide a PR. IMHO.

Reply:
> @amizurov @normanmaurer
> After the test found that the client port is not detected by the server

I think we can handle index.html differently from resources (js, cs) with some simple templating. WDYT @normanmaurer

Reply:
At the end I think this is just an example and we should not over-complicate things.

Reply:
@normanmaurer 
If the example can be close to the actual application scenario, the integration cost can be reduced;
I added some supplements to the above example to see if integration is needed;
![image](https://user-images.githubusercontent.com/22343834/78561368-2ca2c800-784a-11ea-9737-d75e0ae17904.png)
![image](https://user-images.githubusercontent.com/22343834/78561475-6378de00-784a-11ea-8bae-237033a447c4.png)